### DISCUSSION TITLE
Runtime e Sdk's versões

### DISCUSSION THREAD

**Discussion by lifreir** [balta-io/2801#29] [Perguntas e Respostas] [answered]
Na aula que fala de runtime, não entendi muito bem uma coisa, se existem várias versões do sdk, o Runtime também possui várias versões?  Se sim elas podem ser instaladas no mesmo computador?  Ou eu preciso ter a runtime correta para executar uma determinada versão do sdk?
Ise puderem me ajudar a entender agradeço.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @lifreir, como vai?
Sim, existem várias versões e elas podem ser instaladas lado-a-lado no mesmo computador sem problemas <3
